test/connectionCredentials.test.ts

```diff
@@ -78,6 +78,51 @@ suite('ConnectionCredentials Tests', () => {
             connectionStore.object);
     }
 
+    function ensureRequestAndSavePassword(emptyPassword: boolean): (done: MochaDone) => void {
+
+        return (done: MochaDone) => {
+            // Setup Profile Information to have savePassword off and blank
```

Comment:
It looks like the original intention of the test was to have save password as on. The comment probably wasn't updated after a copy-paste. Will change